### ISSUE TITLE
Perturbation bias correction 

### ISSUE THREAD

**Issue by LIS-navari** [NASA-LIS/LISF#1313]
### Bug Description
Perturbation bias correction has already hard coded in the master branch but not in the support/lisf-557ww-7.5.

https://github.com/NASA-LIS/LISF/blob/5bddcb93c61055831cf75b019d8d587d76d034ca/lis/core/LIS_readConfig.F90#L757

### Steps to Replicate
We may need to update the lisf-557ww-7.5 and remove the 
Apply perturbation bias correction: 1 from the lis.config file. 

@jerrywegiel @emkemp

**Comment by emkemp** [comment]
Adding @jvgeiger and @sujayvkumar to this thread.

My opinion is we do not change the support/lisf-557ww-7.5 branch.  Code from this branch has already been delivered to 557 WW, and we try to avoid further changes unless there is a critical bug fix or new feature needed.  Also, 557 WW will be running this code in production and will be using config settings provided by NASA, so it is unlikely changing the code now will have any effect on 557 WW operations.  Let's keep it frozen.

The presence of the bug fix in master will be sufficient for a future LISF 7.6 deliverable.

Other opinions?

**Comment by sujayvkumar** [comment]
I agree that we don't need to rush this update as long as the configuration entry is correct.

**Comment by emkemp** [comment]
This is moot.  Closing the issue.
